data/logs.go

```diff
@@ -24,12 +24,20 @@ type Event struct {
 	Order      int      `json:"order"`
 }
 
+// StatusInfo holds the fields for the transaction status
```

Comment:
We should move these in mx-chain-core-go since you are referring a new mx-chain-core-go version. They've become high level definitions

Reply:
moved.